web/docs/getting-started.md

```diff
@@ -11,10 +11,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ## 1. Requirements
 
-### Node.js >= 12.18.0
+### Node.js compatible with 16.0.0
+```shell-session
+node -v  # must be something like 16.x.x
+```
 
+### NPM compatible with 8.0.0
 ```shell-session
-node -v  # >= v12.18.0
+npm -v  # must be something like 8.x.x
```

Comment:
@faassen Is this what you had in mind? We now only allow Node versions that look like `16.x.x`, meaning that 17.x.x. and higher won't work. Same goes for NPM.

Reply:
As we discussed, I think this is ok. We had `>= 12` and that became problematic, so let's try to use more conservative approach and see how that goes for us.